.ci/ci_cache_image.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+function handle_error {
+	local exit_code="${?}"
+	local line_number="${1:-}"
+	echo "Failed at $line_number: ${BASH_COMMAND}"
+	exit "${exit_code}"
+}
+trap 'handle_error $LINENO' ERR
+
+WORKSPACE=${WORKSPACE:-$(pwd)}
+kata_dir="/usr/share/kata-containers/"
+
+
+cache_build() {
+        type="${1}"
+
+        if [ "${type}" == "image" ]; then
+                link="${kata_dir}/kata-containers.img"
+        else
+                link="/usr/share/kata-containers/kata-containers-${type}.img"
+        fi
+        path=$(readlink -f ${link})
+        echo $(basename "${path}") >  "${WORKSPACE}/artifacts/latest-${type}"
+        sudo sudo cp  "${path}" "/usr/share/kata-containers/osbuilder-${type}.yaml"  "${WORKSPACE}/artifacts"
```

Comment:
- Double `sudo` (I hereby select "*Sussudio*" by Phil Collins as the theme tune for this PR as that's the closest I could get :smile:)
- `${kata_dir}`